tests/benchdnn/graph/input_displacer.cpp

```diff
@@ -264,16 +343,16 @@ int partition_data_displacer_t::displace_input_data(
         return FAIL;
     }
 
-    if (quantize_displace_.find(lt_id) == quantize_displace_.end()) {
+    if (displace_args_.find(lt_id) == displace_args_.end()) {
         // no need to displace the data of this tensor
         return OK;
     }
-    const displace_t &displace = quantize_displace_.at(lt_id);
-
-    const auto &main_op = ::std::get<0>(displace);
-    auto main_op_offset = ::std::get<1>(displace);
-    auto tensor = ::std::get<2>(displace);
-    const auto filling_type = ::std::get<3>(displace);
+    const displace_args_t &d_args = displace_args_.at(lt_id);
+    const auto &main_op = d_args.main_op_;
+    auto main_op_offset = d_args.main_op_offset_;
+    auto tensor = d_args.tensor_;
+    auto &fill_cfg = d_args.fill_cfg_;
```

Comment:
Two members above seem to fit `const` category as well.
```suggestion
    const auto &fill_cfg = d_args.fill_cfg_;
```